example/visionos/Podfile

```diff
@@ -0,0 +1,16 @@
+require_relative '../node_modules/react-native-test-app/visionos/test_app'
+
+workspace 'Example.xcworkspace'
+
+platform :visionos, '1.0'
```

Comment:
Is this required?

Reply:
Yes, the main way cocoapods knows to set the platform in this case.

Reply:
Then would it be part of the template as well?

Reply:
It seems I am wrong. Latest iteration works without this line locally.